basm.go

```diff
@@ -21,6 +21,11 @@ func Log(msg string) {
 	hostFuncBufferLog(msg)
 }
 
+// LogToHost writes a message to the host's console.
```

Comment:
Adding this function, so that I can do logging to host in the local server started through `bky-as`

Reply:
Fix comment, this comment is incorrect and should be updated.